lms/djangoapps/api_manager/sessions/views.py

```diff
@@ -120,6 +121,10 @@ def post(self, request):
                     response_data['uri'] = '{}/{}'.format(base_uri, new_session.session_key)
                     response_status = status.HTTP_201_CREATED
 
+                    # generate a CSRF tokens for any web clients that may need to
+                    # call into the LMS via Ajax (for example Notifications)
+                    response_data['csrftoken'] = RequestContext(request, {}).get('csrf_token')
```

Comment:
LGTM -- CSRF Tokens weren't a necessity for the original server API -- we did look into supporting them for the API workflows but ultimately decided they weren't required due to the integration scenario (ie, server-to-server)